src/utils.ts

```diff
@@ -202,3 +202,6 @@ export function isDelayedTransitionAction(
       return false;
   }
 }
+
+export const isInputLikeElement = (el: HTMLElement) =>
+  el.tagName === 'INPUT' || el.tagName === 'TEXTAREA' || el.isContentEditable;
```

Comment:
TIL I've learned that there is `.isContentEditable` property that you can check and it works for all elements within `[contendeditable]`